Makefile

```diff
@@ -90,30 +97,31 @@ precommit: cover gofmt lint test
 $(COMPILER_BIN) $(RUNNER_BIN) $(TOOL_BINS): build/bin/%: tools/%
 	@mkdir -p build/bin
 	@cp -f $< $@
+	@sed -i '' '1s@/usr/bin/env python@$(PYTHON_BIN)@' $@
```

Comment:
@trotterdylan, after some tries, on debian, sed -i'' without space only works and 
on OSX, sed -i '' with space only works for me.